### DISCUSSION TITLE
用electron把项目打包成可执行文件后，安装的时候 360安全卫士会报有程序试图修改关键程序DLL文件。 然后定位到d3dcompiler_47.dll  有解决的办法吗？

### DISCUSSION THREAD

**Discussion by lixinyao0** [alex8088/electron-vite#324] [Q&A] [answered]
用electron把项目打包成可执行文件后，安装的时候 360安全卫士会报有程序试图修改关键程序DLL文件。 然后定位到d3dcompiler_47.dll  有解决的办法吗？
![image](https://github.com/alex8088/electron-vite/assets/61485160/46b1fc99-2355-46cc-887c-34d1966ed210)

**Comment by alex8088** [comment, marked as answer]
正常，想要解决就需要签名，签名需要购买证书。

**Reply by FindChenLinJie** [reply]
签名了，也同样报这个问题，请问还可以怎么解决

**Comment by huhuime** [comment]
这个文件是有微软签名的，但是比微软更新的版本更低，理论上系统里有的话不用打包这个（不需要运行在win7sp1以下的版本应该是不用的）
